crawl-ref/source/cloud.cc

```diff
@@ -1814,3 +1816,105 @@ void end_still_winds()
     env.level_state &= ~LSTATE_STILL_WINDS;
     mpr("The air resumes its normal movements.");
 }
+
+void manage_cloud_generators() {
+    if (env.special_clouds == 1) {
```

Comment:
Code style: all braces should go on their own line for function definitions, `if`s, `for`s, `enum class`es, etc. There are at least thirteen instances where this needs to be changed; I am only going to list the exceptions.